docs_nnx/conf.py

```diff
@@ -140,7 +140,7 @@
 
 # -- Options for myst ----------------------------------------------
 # uncomment line below to avoid running notebooks during development
-# nb_execution_mode = 'off'
+nb_execution_mode = os.environ.get("NB_EXECUTION_MODE", 'off')
```

Comment:
This helps to build the docs without uncommenting this line and skipping notebooks execution:
```bash
NB_EXECUTION_MODE=off make html
```